modules/wsl-distro.nix

```diff
@@ -207,14 +207,25 @@ in
 
     # require people to use lib.mkForce to make it harder to brick their installation
     wsl = {
-      populateBin = true;
+      populateBin = !config.services.envfs.enable;
```

Comment:
This should use lib.mkIf and set the ture Boolean literal to prevent any unforseen condition fuckups.

Reply:
Updated!